tests/utils/bgp_convergence_config.py

```diff
@@ -1,7 +1,3 @@
-import pytest
-
-
-@pytest.fixture
 def bgp_convergence_config(api, utils):
```

Comment:
Do not move it to utils. Move it back to convergence folder. We should keep fixtures specific to a given test scenario in concerned directories, just like sonic-mgmt.